cpp/src/parquet/arrow/writer.cc

```diff
@@ -34,6 +34,7 @@
 
 #include "arrow/util/logging.h"
 
+#include "parquet/arrow/reader.h"
```

Comment:
I think this relates to my comment on the additional parameter in `FromParquetSchema`.  Having the writer depend on the reader at first-glance appears as an anti-pattern.

Reply:
Will clean up includes, this was about getting access to ArrowReaderProperties